components/resources/ai_chat_prompts.grdp

```diff
@@ -44,7 +44,8 @@ Here is the conversational history (between the user and you) prior to the quest
 <message name="IDS_AI_CHAT_CLAUDE_QUESTION_PROMPT_SEGMENT" translateable="false">
 Propose up to 3 very short questions that a reader may ask about the content. Consider intriguing or unusual elements of the content, or structurally important. Separate each suggested question with the character "|". End the question list with a &lt;/response&gt; tag.</message>
 
-<message name="IDS_AI_CHAT_QUESTION_SUMMARIZE_VIDEO_BULLETS" translateable="false">Provide a concise list of up to 6 bullets of the most important points of the video.</message>
+<message name="IDS_AI_CHAT_QUESTION_SUMMARIZE_PAGE" translateable="false">Provide a concise list of up to 6 bullets on the most important points of the page.</message>
```

Comment:
Is this meant to make the summaries' formats more consistent? Before we were just using "Summarise the page", correct?

One possible suggestion is that we currently treat the page as an article (<article> tags), so referring to it as an article might help. We could also switch that article tag with a page tag since our goal seems to be summarizing/answering on more general types of content, and switch all the language to "page" to be consistent

Reply:
Well, yes I was fixing this issue where we're translating this part of the prompt, but then was wondering why we would differ so much from the video summary request. But perhaps I should just fix this issue whereby we don't translate, but keep it as "Summarize this page"?

Unless we think there's definitely an improvement to changing `<article` to `<page`, and/or using this more descriptive request for summary?

Now I'm wondering - was there actually value to keeping this part of the prompt translated to the user's UI language? This is the part of the prompt which pretends to be the user, so maybe we should translate those parts? Not sure if that's reliable. We also don't translate the generated suggested questions. And if they're not in a nicely supported language then it could have weird results. Do you agree that we should stop translating this part of the prompt?

Reply:
I've changed to `<page` everywhere, and it seems to be pleasant results in both llama and claude